bno055.h

```diff
@@ -521,6 +521,7 @@ struct bno055_t
     BNO055_WR_FUNC_PTR; /**< bus write function pointer */
     BNO055_RD_FUNC_PTR; /**<bus read function pointer */
     void (*delay_msec)(BNO055_MDELAY_DATA_TYPE); /**< delay function pointer */
+    void *dev_handle; /**< device function pointer */

```

Comment:
can we make this 'user_data'?

Reply:
and clarify that its used to hold context, like /**< User data pointer */

Reply:
changed to user_data